### DISCUSSION TITLE
Issue with Importing SCSS File in Next.js Project

### DISCUSSION THREAD

**Discussion by hajpepe** [vercel/next.js#55204] [Help]
### Summary

Yo,

Hope you're chillin', my friend! So, here's the deal – I'm cookin' up a project in Next.js, and I'm hittin' a little hiccup with my SCSS import game. It's all good, though, 'cause I know you got that knowledge!

Issue at Hand: ./src/app/globals.css Module not found: Can't resolve './globals.scss'

Project Setup: Peep this:

./src/app/globals.css
./src/app/globals.scss
In my globals.css, I'm tryin' to drop in globals.scss like this:
@import './globals.scss';

I've double-checked the file's existence, the path, and the file extension – all seems legit. But still, I'm gettin' this error, and I'm like, "What's crackin'?"

I'd be super grateful if you could drop some wisdom, maybe some tweaks or tricks I ain't hip to yet. If there's any specific magic spells or dependencies I gotta throw in the mix, holla atcha boy!

Keep it real and stay lit,

**Comment by devjiwonchoi** [comment]
I bet you'd need to check on this: https://nextjs.org/docs/app/building-your-application/styling/sass

Most likely on `next.config.js`:

```js
const path = require('path')
 
module.exports = {
  sassOptions: {
    includePaths: [path.join(__dirname, 'styles')],
  },
}
```

**Reply by devjiwonchoi** [reply]
Where are you trying to import it? `./` is the base of your importing file, for example on `navbar.js`, you need to set as `../globals.scss`. Note that you can use `import alias` as `@/globals.scss`.

**Reply by hajpepe** [reply]
i want override the bootstrap but as u see 
i have 
module.exports = {
  sassOptions: {
    includePaths: [path.join(__dirname, 'styles')],
  },
} 
oh look at there dirname, 'styles') so we  d'have styles dir :D

**Reply by hajpepe** [reply]
may that's problem

**Reply by hajpepe** [reply]
i want import global.scss into global.css cause GPT told me if u change bootstrap variables but not works do that ,

my goal is  override stroke color form bootstrap

globals.scss :
@import "bootstrap/scss/bootstrap";

// Override the Bootstrap border-bottom color
$border-bottom: (
  property: border-bottom,
  values: (
    null: 1px solid #757575, // Change this color to #757575
    0: 0,
  )
);
